frontend/src/metabase/lib/utils.js

```diff
@@ -1,5 +1,10 @@
 import generatePassword from "password-generator";
 
+function s4() {
+    return Math.floor((1 + Math.random()) * 0x10000)
+      .toString(16)
+      .substring(1);
+}
```

Comment:
`s4` is a pretty confusing function name 😢 . How about something more descriptive like `rand4CharHexStr`?